mobile/library/common/config/templates.h

```diff
@@ -102,3 +102,13 @@ extern const char* default_cert_validation_context_template;
  * chain.
  */
 extern const char* platform_cert_validation_context_template;
+
+/**
+ * Config template for an RTDS layer
+ */
+extern const char* rtds_layer_insert;
+
+/**
+ * Config template for an XDS cluster
+ */
+extern const char* xds_cluster_insert;
```

Comment:
i think we probably want to change this name to `cluster_insert` and the builder API to `addCluster`